Files/DataModels/SidebarPinnedModel.cs

```diff
@@ -278,6 +278,21 @@ public async Task AddItemToSidebarAsync(string path)
             }
         }
 
+        /// <summary>
+        /// Adds the item to sidebar asynchronous.
+        /// </summary>
+        /// <param name="section">The section.</param>
+        public async Task AddItemToSidebarAsync(LocationItem section)
```

Comment:
This can be just `private void AddItemToSidebarAsync(LocationItem section)` there in nothing async in here